includes/utilities.inc

```diff
@@ -409,11 +418,18 @@ function islandora_solr_islandora_basic_collection_backend_callable($collection_
  *   Min wordsafe length.
  * @param string $separator
  *   A separator to use for display output.
+ * @param array $link_options
+ *   An array of link options including:
+ *   - 'link_to_object' : (bool) Include link to object.
+ *   - 'link_to_search' : (bool) Include link to search results.
+ *   - 'field' : (SOLR Result Field) Result field.
+ *   - 'url' : (SOLR object results object_url) objects result url.
+ *   - 'options' : link to object l() options param.
  *
  * @return string
  *   The updated display values.
  */
-function islandora_solr_truncate_field_display($display_values, $max_length, $add_ellipsis, $word_safe, $wordsafe_length, $separator) {
+function islandora_solr_truncate_field_display($display_values, $max_length, $add_ellipsis, $word_safe, $wordsafe_length, $separator, $link_options) {
```

Comment:
If this signature is being changed [this](https://github.com/Islandora/islandora_solr_metadata/blob/7.x/theme/theme.inc#L126) and any other places where it gets references need to be updated.